Classes/Login/LoginViewController.swift

```diff
@@ -53,6 +57,11 @@ final class LoginViewController: UITableViewController, UITextFieldDelegate {
             }
         }
     }
+    
+    @IBAction func onCancel(_ sender: Any) {
+        guard !isInitialLogin else { fatalError("Should not be able to cancel during initial login") }
+        handleResult(.cancelled, login: "")
```

Comment:
Not sure about this - we don't use (or have a) "login" when cancelling, but just passing `""` feels wrong. Making it optional is a hassle, but might better reflect the possible state?